config/config.default.js

```diff
@@ -17,9 +17,11 @@ module.exports = () => {
    * @property {Object} xssProtection - 是否开启IE8的XSS Filter，默认开启。
    * @property {Object} csp - csp安全配置。
    * @property {Array} domainWhiteList - 安全跳转白名单
+   * @property {Array} protocolWhiteList - 协议白名单
    */
   exports.security = {
     domainWhiteList: [],
+    protocolWhiteList: [],
```

Comment:
If there's such shared whiteList, it can be configured as two levels: `config.security.whiteList` and `config.helper.shtml.whiteLIst`. The latter one have high priority. So this approach guarantees backward compatibility when needed.